blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/RunContainerImpl.java

```diff
@@ -49,6 +56,20 @@ public BlueRun get(String name) {
                 }
             }
             if (run == null) {
+                // JENKINS-38540 - To make this consistent with the activity API, check the queue
+                String queuedRun = findRunInQueue(name);
+                if (queuedRun != null) {
+                    try {
+                        StaplerResponse rsp = Stapler.getCurrentResponse();
+                        // Send a 302, temporary redirect. substring to fix double slash
+                        rsp.sendRedirect(Links.ensureTrailingSlash(Jenkins.getInstance().getRootUrl() + pipeline.getLink().toString().substring(1) + "queue/" + queuedRun));
```

Comment:
They're a bit different, but very similar other than one of them redirects to `"runs/" + runId` while the other redirects to `"queue/" + queuedRun` and calls `Links.ensureTrailingSlash`. Not a biggie.